CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# 10.1.0
+
+* Adds backwards-compatible support for Rails 7.2
+
+# 10.0.0
+
+* Adds support for Rails 7.1
+* Versions of Rails older than 7.1 are no longer supported
```

Comment:
I think versions up to v5 have descriptions in the releases themselves but I don't feel like spelunking into the versions between then and now to find out what happened in each new release